client/src/App.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import LineChart from './LineChart';
+
+export default class FetchStockQuotes extends Component {
+  
+  state = {
+    loading: true,
+    data: null
+  };
+
+  async componentDidMount(){
+    const url = 'http://localhost:3000/AAPL';
+    const response = await fetch(url);
+    const data = await response.json();
+  
+    this.setState({
+      loading: false,
```

Comment:
Having a "loading" property could definitely be useful for a client waiting for a page to load.